website/docs/commands/publishing/publish.md

```diff
@@ -117,6 +117,9 @@ handled by either
 - the [Bouncy Castle library](https://www.bouncycastle.org) (default, recommended)
 - the local `gpg` binary on your machine
 
+A signing mechanism will be chosen based on options and directives specified,
+it can also be overriden with `--signer` with one of those values: `bc`, `gpg` or `none`.
```

Comment:
Can you explain the meanings of the values `bc` and `gpg`? Also, what happens when the none option is selected?